packages/cli/src/environments/variables/variables.service.ts

```diff
@@ -1,20 +1,53 @@
 import type { Variables } from '@db/entities/Variables';
-import { collections } from '@/Db';
+import { CacheService } from '@/services/cache.service';
+import Container, { Service } from 'typedi';
+import { VariablesRepository } from '@/databases/repositories';
+import type { DeepPartial } from 'typeorm';
 
+@Service()
 export class VariablesService {
-	static async getAll(): Promise<Variables[]> {
-		return collections.Variables.find();
+	constructor(
+		protected cacheService: CacheService,
+		protected variablesRepository: VariablesRepository,
+	) {}
+
+	async getAll(): Promise<Variables[]> {
```

Comment:
I think we discussed using `-Cached` to mark these?

Reply:
true!